com.blockstream.Green.yml

```diff
@@ -0,0 +1,39 @@
+app-id: com.blockstream.Green
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: green
+modules: 
+  - name: green
+    buildsystem: simple
+    build-commands:
+      - bsdtar -xf BlockstreamGreen_Linux_x86_64.tar.gz
+      - install -D green ${FLATPAK_DEST}/bin/green
+      - desktop-file-install --dir ${FLATPAK_DEST}/share/applications com.blockstream.Green.desktop
+      - install -Dm644 green.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.blockstream.Green.png
+      - install -Dm644 com.blockstream.Green.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.blockstream.Green.metainfo.xml
+
+    sources:
+      - type: file
+        url: https://github.com/Blockstream/green_qt/releases/download/release_1.2.0/BlockstreamGreen_Linux_x86_64.tar.gz
```

Comment:
so how is this gonna work on aarch64?

Reply:
It won't. We only provide binaries for x86_64 currently.

Reply:
you can build from source. As you currently still try to make an aarch64 flatpak

Reply:
Aarch64 support isn't production ready yet.

Reply:
you should still build from source